### ISSUE TITLE
Mitglieder werden nicht angezeigt

### ISSUE THREAD

**Issue by f1shl** [zoomyboy/adrema#3]
Hi,

die Installation verlief auf Mac Apple Silicon reibungslos bei mir.
Leider werden jedoch keine Mitglieder angezeigt. In der initialen Preview habe ich alle Mitglieder meines Stammes gesehen, aber synchronisiert wird dann doch nichts. Kann das an fehlenden Rechten liegen?

VG
Fischl

**Comment by zoomyboy** [comment]
Ja, das kann an fehlerhaften Rechten liegen. Du solltest Lese-UND Schreibrechte auf deine Gruppierung haben.

Außerdem solltest du checken ob der horizon container läuft. Sonst sende mir mal per Mail deine Docker Logs.
